Compilation error when disabling pubkey authentication (DROPBEAR_SVR_PUBKEY_AUTH)

Using latest dropbear 2022.82, with only one option defined in localoptions.h:
```
#define DROPBEAR_SVR_PUBKEY_AUTH 0
```

The build fails with the following error message:
```
In file included from session.h:32:0,
                 from svr-chansession.c:28:
auth.h: In function 'svr_pubkey_set_forced_command':
auth.h:57:67: warning: unused parameter 'chansess' [-Wunused-parameter]
 static inline void svr_pubkey_set_forced_command(struct ChanSess *chansess) { }
                                                                   ^
svr-chansession.c: In function 'execchild':
svr-chansession.c:1043:26: error: 'struct AuthState' has no member named 'pubkey_info'
         if (ses.authstate.pubkey_info != NULL) {
                          ^
svr-chansession.c:1044:58: error: 'struct AuthState' has no member named 'pubkey_info'
                 addnewvar("SSH_PUBKEYINFO", ses.authstate.pubkey_info);
                                                          ^
make: *** [svr-chansession.o] Error 1
```

--- a/svr-chansession.c
+++ b/svr-chansession.c
@@ -1040,9 +1040,11 @@ static void execchild(const void *user_data) {
 	if (chansess->original_command) {
 		addnewvar("SSH_ORIGINAL_COMMAND", chansess->original_command);
 	}
+#if DROPBEAR_SVR_PUBKEY_OPTIONS_BUILT
         if (ses.authstate.pubkey_info != NULL) {
-                addnewvar("SSH_PUBKEYINFO", ses.authstate.pubkey_info);
+            addnewvar("SSH_PUBKEYINFO", ses.authstate.pubkey_info);
         }
+#endif
 
 	/* change directory */
 	if (chdir(ses.authstate.pw_dir) < 0) {

--- a/svr-chansession.c
+++ b/svr-chansession.c
@@ -1041,9 +1041,9 @@ static void execchild(const void *user_data) {
 		addnewvar("SSH_ORIGINAL_COMMAND", chansess->original_command);
 	}
 #if DROPBEAR_SVR_PUBKEY_OPTIONS_BUILT
-        if (ses.authstate.pubkey_info != NULL) {
-            addnewvar("SSH_PUBKEYINFO", ses.authstate.pubkey_info);
-        }
+	if (ses.authstate.pubkey_info != NULL) {
+		addnewvar("SSH_PUBKEYINFO", ses.authstate.pubkey_info);
+	}
 #endif
 
 	/* change directory */

